### PR TITLE
Modified dependency

### PR DESCRIPTION
**Pull request by routiful**
This commits are related to https://github.com/ROBOTIS-GIT/turtlebot3/issues/481

### CODE REVIEW

File: turtlebot3_gazebo/launch/robot_state_publisher.launch.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# /*******************************************************************************
```
Comment (robotpilot) [inline turtlebot3_gazebo/launch/robot_state_publisher.launch.py]: Please change to python file comment type.

File: turtlebot3_gazebo/launch/robot_state_publisher.launch.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    TURTLEBOT3_MODEL = os.environ['TURTLEBOT3_MODEL']
+
+    use_sim_time = LaunchConfiguration('use_sim_time', default='false')
+    urdf_file_name = 'turtlebot3_' + TURTLEBOT3_MODEL + '.urdf'
+
+    print("urdf_file_name : {}".format(urdf_file_name))
```
Comment (robotpilot) [inline turtlebot3_gazebo/launch/robot_state_publisher.launch.py]: Please change to quotes instead of double-quotes.

Comment (robotpilot) [reply turtlebot3_gazebo/launch/robot_state_publisher.launch.py]: If possible, please use the info function.

File: turtlebot3_gazebo/launch/robot_state_publisher.launch.py (hunk 1 above)
Comment (routiful) [reply turtlebot3_gazebo/launch/robot_state_publisher.launch.py]: I've modified Thanks!

File: turtlebot3_gazebo/launch/robot_state_publisher.launch.py (hunk 2 above)
Comment (routiful) [reply turtlebot3_gazebo/launch/robot_state_publisher.launch.py]: I've modified Thanks!

File: turtlebot3_gazebo/CMakeLists.txt [hunk 3]
```diff
@@ -18,7 +18,7 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(gazebo_ros_pkgs REQUIRED)
-find_package(turtlebot3 REQUIRED)
+find_package(turtlebot3_description REQUIRED)
```
Comment (mikaelarguedas) [inline turtlebot3_gazebo/CMakeLists.txt]: Is this line necessary? It seems that turtlebot3_description is used only in launchfiles and is not used by any of the cmake logic

File: turtlebot3_gazebo/CMakeLists.txt [hunk 4]
```diff
@@ -33,4 +33,4 @@ install(DIRECTORY models rviz worlds launch
 ament_export_dependencies(rclcpp)
 ament_export_dependencies(gazebo_ros_pkgs)
 ament_export_dependencies(turtlebot3)
```
Comment (mikaelarguedas) [inline turtlebot3_gazebo/CMakeLists.txt]: This line should be updated (or removed as suggested in comment above)

File: turtlebot3_gazebo/package.xml [hunk 5]
```diff
@@ -17,7 +17,7 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <depend>rclcpp</depend>
   <depend>gazebo_ros_pkgs</depend>
-  <depend>turtlebot3</depend>
+  <depend>turtlebot3_description</depend>
```
Comment (mikaelarguedas) [inline turtlebot3_gazebo/package.xml]: As this is only a runtime dependency. Using `exec_depend` would be more appropriate

File: turtlebot3_gazebo/CMakeLists.txt (hunk 3 above)
Comment (routiful) [reply turtlebot3_gazebo/CMakeLists.txt]: I agreed. I've deleted.

File: turtlebot3_gazebo/CMakeLists.txt (hunk 4 above)
Comment (routiful) [reply turtlebot3_gazebo/CMakeLists.txt]: My mistake. I've deleted.

**Comment by mikaelarguedas** [comment]
Great thanks! :+1:

**Comment by routiful** [comment]
@robotpilot Ping. Please merge this request.

**Pull request merged**